module1/00_intro.en.md

```diff
@@ -0,0 +1,122 @@
+<!-- .slide: data-background="#111111" -->
+
+# Object oriented programming
+
+<a href="https://coders.school">
+    <img width="500" data-src="../coders_school_logo.png" alt="Coders School" class="plain">
+</a>
+
+___
+
+## SHM (Sea Trade Simulator)
+
+<img height="500" data-src="img/statek.png" src="https://github.com/coders-school/object-oriented-programming/raw/master/module1/img/statek.png" alt="Statek" class="plain">
+
+___
+<!-- .slide: data-background="#eeeeee" -->
+
+## SHM - UML diagram
+
+<a href="https://raw.githubusercontent.com/coders-school/object-oriented-programming/master/shm-uml.svg">
+    <img height="500" data-src="../shm-uml.svg" src="https://raw.githubusercontent.com/coders-school/object-oriented-programming/master/shm-uml.svg" alt="Diagram klas - SHM" class="plain">
+</a>
```

Comment:
I think that either abbrevation should be changed to STS or there should explanation something like |SHM(pol. Symulator handlu morskiego) - Sea Trade Simulator"

Reply:
Yeah, putting polish version should clear things out. Thanks 🙂